test/types/type_variables/ferguson/unnecessary-type-construction.good

```diff
@@ -0,0 +1,27 @@
+unnecessary-type-construction.chpl:3: warning: unnecessary type construction call
+unnecessary-type-construction.chpl:1: note: 'R' has no generic fields
+unnecessary-type-construction.chpl:3: note: remove the ()
```

Comment:
Nitpicky, but I'd put the parens into single quotes `'()'` to further demarcate them as being "not normal English" (vs. "our error message has a bug and forgot to print something else here")